### ISSUE TITLE
Fix color contrast on email newsletter Subscribe button

### ISSUE THREAD

**Issue by star-szr** [drupaldiversity/administration#85]
#fff on #39B5F2 doesn't meet WCAG AA or AAA, let's change it.

http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=39B5F2

**Comment by cleverington** [comment]
It almost looks like it expects black text to match that color blue.

I would say a better option (then darkening the text) would be to darkening 

I tried like.... 10 different approaches and I gave up.

Thoughts on adding a third color, like purple or something?

**Comment by sugaroverflow** [comment]
@cleverington has this been discussed/decided on? I want to move this issue to drupaldiversity/drupaldiversity.github.io instead of this queue (since this admin queue is moving to d.o)

**Comment by cleverington** [comment]
@sugaroverflow This task can be one for moving. The discussion you see is the contents of the discussion there has been.  Meeting WCAG and AAA should be a _requirement_ for the DD&I group, so migrating it as an issue would be a good idea.

**Comment by sugaroverflow** [comment]
Thank you @cleverington! 

This has been moved over to the website issue queue, thank you! https://github.com/drupaldiversity/drupaldiversity.github.io/issues/40
